### PR TITLE
Test parse stories information

### PR DESCRIPTION
**Pull request by gafmn**

### CODE REVIEW

File: tests/test_parse_data.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    for i in range(2):
+        stub_data = {
+                'title': 'Test' + str(i),
+                'score': 1234,
+                'url': 'there is no link'
+        }
+        yield stub_data
+
+
+def test_build_stories_info():
```
Comment (ilov3) [inline tests/test_parse_data.py:34]: Could you please add a doc-string to each test definition, which will highlight steps and the purpose of each test?

File: tests/test_parse_data.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import inspect
+import sys
+
+currentdir = os.path.dirname(   # type: ignore
+    os.path.abspath(
+        inspect.getfile(inspect.currentframe())     # type: ignore
+    )
+)
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
```
Comment (ilov3) [inline tests/test_parse_data.py]: I think it make sense to separate this code into its own module as it described here - https://docs.python-guide.org/writing/structure/#test-suite (the part about "import context")

File: tests/test_parse_data.py (hunk 1 above)
Comment (gafmn) [reply tests/test_parse_data.py:34]: Okay, I will add it

File: tests/test_parse_data.py (hunk 2 above)
Comment (gafmn) [reply tests/test_parse_data.py]: Fixed it

**Pull request merged**